lang/pt.php

```diff
@@ -91,7 +91,7 @@
 $monetizainfo="Leia como funciona";
 $pesodelvoto="Peso do voto";
 $poderdevoto="Valor do voto";
-$contenidoexclusivo="Por favor, não publique conteúdo que já tenha sido publicado anteriormente em outras plataformas. Obrigado.";
+$contenidoexclusivo="Por favor, não publique conteúdo que já tenha sido publicado anteriormente se for se inscrever no #cashfortricks. Obrigado.";

```

Comment:
Changes of meaning must be done in all languages.

Reply:
Actually , most of apps that I worked with had different translations per language, according with the goals in the regions. 

But we should indeed change this to #cashtricks only in all the languages, but one day we might wanna say different stuff between languages.